activerecord/lib/active_record/persistence.rb

```diff
@@ -292,7 +293,7 @@ def update_column(name, value)
     #   user.update_columns(last_request_at: Time.current)
     #
     # This is the fastest way to update attributes because it goes straight to
-    # the database, but take into account that in consequence the regular update
+    # the database, but takes into account that in consequence the regular update
```

Comment:
I think `take` was right here. As I read it, it refers to the reader to "take something into account".